src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Financial/RoatpFinancialApplicationViewModel.cs

```diff
@@ -71,7 +100,46 @@ private void SetupGradeAndFinancialDueDate(FinancialReviewDetails financialRevie
                         break;
                 }
             }
+
+            if(FinancialReviewDetails.SelectedGrade == FinancialApplicationSelectedGrade.Inadequate)
+            {
+                InadequateComments = FinancialReviewDetails.Comments;
+            }
+            else if (FinancialReviewDetails.SelectedGrade == FinancialApplicationSelectedGrade.Clarification)
+            {
+                ClarificationComments = FinancialReviewDetails.Comments;
+            }
         }
-    }
 
+        private void SetupDeclaredInApplication(RoatpApplyData applyData)
+        {
+            const int fhaSequenceNo = 2;
```

Comment:
Can RoatpQnaConstants.RoatpSequences.FinancialEvidence be used here?

Reply:
done